### ISSUE TITLE
Google cloud dataproc not recognizing 'sqlContext' command

### ISSUE THREAD

**Issue by Indra2009** [spotify/spark-bigquery#58]
I am a newbie in Spark & Scala. I ran this package in local SBT and everything worked fine. 
But now I am trying to do the connection in Google cloud dataproc and executed the command:

spark-shell --packages com.spotify:spark-bigquery_2.10:0.2.0

But after that when I am trying to run the following command at scala prompt:
val table = sqlContext.bigQueryTable("bigquery-public-data:samples.shakespeare")
It can't recognize 'sqlContext' command.
Am I missing something?

**Comment by vatshy** [comment]
Did you import types?
` import com.spotify.spark.bigquery._`

**Comment by Indra2009** [comment]
Yes after spark-shell command I imported the package. And after that I am trying to run the sqlcontext command. but it can't recognize it.

**Comment by randoFelipe** [comment]
any help with this?
